### ISSUE TITLE
Heatmap is missing color scale 

### ISSUE THREAD

**Issue by mehukatti212** [hep7agon/cfh-ui#85]
Heatmap view is missing color scale information. i.e.

red = 10 feedbacks?
yellow = 5?
blue  = 1?

so the scale is required that the user understands the coloring on the map.

**Comment by mehukatti212** [comment]
http://www.hel.fi/palvelukartta/ <-- check väestötiedot and how it scales the map.

**Comment by sevesalm** [comment]
Yeah, a legend would be good. And the current implementation has some interesting way to scale the heatmap while the user zooms the map. I was also thinking that a fixed heatmap would maybe be better. But we'll look into it.

**Comment by mehukatti212** [comment]
also its no secret that there are feedbacks in the map. its all public data. so it should be possible that the user can view feedbacks in an area.
